src/model/MqttPublishAction.ts

```diff
@@ -35,6 +35,8 @@ class MqttPublishAction implements Action {
 
     invokeEvenOnFail = false;
 
+    allowFailure = false;
```

Comment:
If I am not mistaken, allowFailure does not have any effect on MQTT actions and websocket actions (because failures are simply ignored for them anyways). I think the correct way forward would be to make this new configuration apply to those actions as well (for backwards compatibility, allowFailure should default to true for these axtions).

If this is absolutely out of scope at the moment, please at least state this explicitly in the description of the pull request.

Reply:
Documentation will be added to the wiki